test/unit.js

```diff
@@ -465,3 +465,22 @@ describe('render', function () {
     });
   });
 });
+
+describe('diff worker', function () {
+  it('should do work in a worker if opts.done is specified', done => {
+    const source = sd.vdom.element('div', null, 'text 1');
+    const destination = sd.vdom.element('div', null, 'text 2');
+    const realDom = sd.vdom.dom(source);
+
+    sd.diff({
+      destination,
+      source,
+      done (instructions) {
```

Comment:
This is the tentatively proposed API: provide a `done` callback and if workers are available it uses them. Still need to patch `render` / `merge` to handle this.